### ISSUE TITLE
Ghost Aggression 

### ISSUE THREAD

**Issue by Ibenartist** [zuffdaddy/phasmohelper#38]
It's been confirmed by the devs that the info the radio guy say about the ghost is just flavour text and actually has no meaning what so ever. So in my pov isn't necessary to have on the phasmo site. I still want to use this site, because the journal itself still doesn't easily show what evidences that is still possible and remaining. And I also enjoy being to easily access the info about the ghost by a click of a button instead of the journal.

**Comment by Hoppi164** [comment]
Hey @Ibenartist 

Could you provide a link to this confirmation by the Dev Team?
I can't seem to find this anywhere..

I've noticed a strong correlation in my games which would suggest the radio guy's hints DO matter..
I've mentioned this assumption in my feature request #41

**Comment by Ibenartist** [comment]
Of course, DKnighter said so in a livestream  with Insym when they were playing together for the first time during the Anniversary. [https://youtu.be/I_EOE9YrlPY?t=1278](url) 21m and 20seconds incase the link doesn't directly take you to the timestamp. 
Personally I rarely experience that the flavour text matches the ghost type. I don't know if he or the other will implement this later however.

**Comment by Ibenartist** [comment]
I guess if some people really do believe in the flavour text have meaning, maybe add a button that could disable it showing if possible? Like you've done with the orbs :)

**Comment by Hoppi164** [comment]
Hey @Ibenartist 

I didn't know that video existed..
Thanks for sending through :)

That makes me doubt a lot of my game-play decisions now...
